azure-pipelines.yml

```diff
@@ -229,11 +229,13 @@ jobs:
                           /p:SignAssetsDir=$(Build.ArtifactStagingDirectory)\Microsoft.Spark.Binaries\Microsoft.Spark.Worker\
                           /p:SignType=$(_SignType)
         msbuildVersion: 15.0
-
-    - task: NuGetCommand@2
+ 
+    - task: DotNetCoreCLI@2
+      displayName: 'Create NuGet packages'
       inputs:
         command: pack
-        packagesToPack: '$(Build.SourcesDirectory)\src\csharp\Microsoft.Spark.nuspec'
+        projects: '$(solution)'
+        arguments: '--no-build --configuration $(buildConfiguration) --output $(Build.ArtifactStagingDirectory)'
```

Comment:
I'm not really sure if --no-build is really needed even if the binaries had been signed

Reply:
I think it is good policy to specify `--no-build` here. We strictly do not want to build/compile the binary while doing `pack`. Better to be explicit IMO.

---
In reply to: [278708264](https://github.com/dotnet/spark/pull/54#discussion_r278708264) [](ancestors = 278708264)

Reply:
Agreed